src/core/scene/a-scene.js

```diff
@@ -547,9 +547,9 @@ module.exports.AScene = registerElement('a-scene', {
         renderer = this.renderer = new THREE.WebGLRenderer(rendererConfig);
         renderer.setPixelRatio(window.devicePixelRatio);
         renderer.sortObjects = false;
-        renderer.vr.setPoseTarget(this.camera);
+        if (this.camera) renderer.vr.setPoseTarget(this.camera.el.object3D);
```

Comment:
Missing curly braces: ` if (this.camera) { renderer.vr.setPoseTarget(this.camera.el.object3D); }`